### PR TITLE
update offchain resolver contracts, add sepolia support

### PR DESCRIPTION
**Pull request by mdtanrikulu**

### CODE REVIEW

**Review by gskril — COMMENTED** [review]
Did you deploy the new OffchainResolver.sol and factory? Would be good to update the addresses (and add Sepolia) [here](https://github.com/ensdomains/ccip-tools/blob/mdt/sepolia/web/components/organisms/deploy/DeployResolver.tsx#L22-L32)

**Comment by mdtanrikulu** [comment]
> Did you deploy the new OffchainResolver.sol and factory? Would be good to update the addresses (and add Sepolia) [here](https://github.com/ensdomains/ccip-tools/blob/mdt/sepolia/web/components/organisms/deploy/DeployResolver.tsx#L22-L32)

done

File: web/components/organisms/deploy/DeployResolver.tsx [hunk 1]
```diff
@@ -138,7 +147,7 @@ export const DeployResolverCard: FC = () => {
             {
                 error && (
                     <p className="text-red-500">
-                        {chainId !== 5 ? 'Only Goerli is supported right now' : error.message}
+                        {(chainId !== 5 || chainId !== 11155111) ? 'Only Goerli and Sepolia is supported right now' : error.message}
```
Comment (gskril) [inline web/components/organisms/deploy/DeployResolver.tsx:150]: Mainnet is supported too! So maybe this should just be `{error.message}`

**Review by gskril — APPROVED** [review]
Left 1 suggestion but otherwise looks good!

We should probs deploy the new resolver + factory contracts to Goerli and Mainnet at some point, but the old versions still work in nearly all cases so it's not urgent.

File: web/components/organisms/deploy/DeployResolver.tsx (hunk 1 above)
Comment (mdtanrikulu) [reply web/components/organisms/deploy/DeployResolver.tsx]: added all supported chains

**Pull request merged**